src/routes/Dashboard/DashboardProducers.js

```diff
@@ -254,9 +257,6 @@ const Producers = () => {
                           alt="logo"
                         />
                         {producer?.owner}
-                        {info.head_block_producer === producer?.owner
-                          ? 'current'
-                          : 'VPLP'}
```

Comment:
@adriexnet  what is VPLP?

Reply:
Sorry it was a text just for testing purposes, we should remove it

Reply:
Thats what I thought , thanks!